tests/e2e/idps_test.go

```diff
@@ -339,8 +338,8 @@ var _ = Describe("TF Test", func() {
 						}
 
 						// login to the cluster using cluster-admin creds
-						username := CON.ClusterAdminUser
```

Comment:
Let's make the alias of the import name as upper string. User `CON` instead of `con`

Reply:
changed to `CON`